integration-tests/capacity/transactions.test.ts

```diff
@@ -70,7 +72,22 @@ describe("Capacity Transactions", function () {
                     assert.fail("should return a DelegationGranted event");
                 }
 
-                assertEvent(chainEvents, "capacity.CapacityWithdrawn");
+                let fee: u128;
+                if (chainEvents["capacity.CapacityWithdrawn"] && 
+                    ExtrinsicHelper.api.events.capacity.CapacityWithdrawn.is(chainEvents["capacity.CapacityWithdrawn"])) 
+                {
+                    fee = chainEvents["capacity.CapacityWithdrawn"].data.amount;
+                }
+                else {
+                    assert.fail("should return a CapacityWithdrawn event");
+                }
+                const reducedCapacity = amountStaked - fee.toBigInt();
+
+                // Check for remaining capacity to be reduced by correct amount
+                const capacityStaked = (await firstValueFrom(ExtrinsicHelper.api.query.capacity.capacityLedger(stakeProviderId))).unwrap();
+                assert.equal(capacityStaked.remainingCapacity, reducedCapacity, "should return a capacityLedger with " + reducedCapacity + " remainingCapacity");
```

Comment:
it would be more readable to use [template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) here: 
```
`should return a capacityLedger with ${reducedCapacity} remainingCapacity`
```

Also it would be less confusing to call the variable remainingCapacity.

Reply:
Good idea, will update.